Plot popup endpoint returns 500 on staging deploy

GET {staging-server}/philadelphia3/plots/11302/
Normal 200 response

GET {staging-server}/philadelphia3/plots/11302/popup
500

Looks like code in `audit.py` is indexing a list before checking if it is empty

Tail of the django request log:

2013-10-30 08:11:52,457 [ERROR] django.request: Internal Server Error: /philadelphia3/plots/11302/popup
Traceback (most recent call last):
  File "/usr/local/otm/env/local/lib/python2.7/site-packages/django/core/handlers/base.py", line 115, in get_response
    response = callback(request, _callback_args, *_callback_kwargs)
  File "/usr/local/otm/app/opentreemap/treemap/util.py", line 138, in wrapper
    return view_fn(request, instance, _args, *_kwargs)
  File "/usr/local/otm/env/local/lib/python2.7/site-packages/django/views/decorators/http.py", line 100, in inner
    res_etag = etag_func(request, _args, *_kwargs)
  File "/usr/local/otm/app/opentreemap/treemap/views.py", line 60, in _plot_hash
    base = get_object_or_404(instance_plots, pk=plot_id).hash
  File "/usr/local/otm/app/opentreemap/treemap/models.py", line 456, in hash
    string_to_hash = super(Plot, self).hash
  File "/usr/local/otm/app/opentreemap/treemap/audit.py", line 947, in hash
    string_to_hash = str(audits[0].pk)
  File "/usr/local/otm/env/local/lib/python2.7/site-packages/django/db/models/query.py", line 215, in __getitem__
    return list(qs)[0]
IndexError: list index out of range

--- a/opentreemap/treemap/audit.py
+++ b/opentreemap/treemap/audit.py
@@ -935,7 +935,7 @@ class Auditable(UserTrackable):
     @property
     def hash(self):
         """ Return a unique hash for this object """
-        # Since this is an audited object each changes will
+        # Since this is an audited object each change will
         # manifest itself in the audit log, essentially keeping
         # a revision id of this instance. Since each primary
         # key will be unique, we can just use that for the hash
@@ -944,7 +944,14 @@ class Auditable(UserTrackable):
                               .filter(model_id=self.pk)\
                               .order_by('-updated')
 
-        string_to_hash = str(audits[0].pk)
+        # Occasionally Auditable objects will have no audit records,
+        # this can happen if it was imported without using save_with_user
+        try:
+            audit_string = str(audits[0].pk)
+        except IndexError:
+            audit_string = 'none'
+
+        string_to_hash = '%s:%s:%s' % (self._model_name, self.pk, audit_string)
 
         return hashlib.md5(string_to_hash).hexdigest()
 

